specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -322,6 +322,142 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/{billingProfileName}/billingInstructions": {
+      "get": {
+        "tags": [
+          "BillingInstructions"
+        ],
+        "operationId": "BillingInstructions_ListByBillingProfile",
+        "description": "Lists the billing instructions by billing profile id.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/billing/"
+        },
+        "x-ms-examples": {
+          "BillingInstructionsListByBillingProfile": {
+            "$ref": "./examples/BillingInstructionsListByBillingProfile.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/billingAccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/billingProfileNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/BillingInstructionListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/{billingProfileName}/billingInstructions/{billingInstructionName}": {
+      "get": {
```

Comment:
Does this support DELETE?

Reply:
Not at this time, no. Eventually we may add support for soft-deletes, but that's out-of-scope for now.

Reply:
Discussed offline. DELETE support will be added when ready, but I'm signing off regardless to unblock testing.